### PR TITLE
Updates to Charter Process

### LINKED ISSUE #29: Timebox the review process for charters and RFCs
There have been a number of proposed charters created at the same time, and it seems to me that adding a time frame for reviews (for both charters and RFCs) is needed. The process of assigning reviewers and soliciting feedback seems to have worked inconsistently too. 

The charter process currently says:
> Feedback is added to the pull request comment thread by community members. 

I would suggest the following modifications for authors:

1. Assign key reviewers, including PMs of impacted/dependant charters
2. Add a deadline for community review comments
3. Send reminders to the #dcp slack channel and to assigned key reviewers directly shortly before the community review deadline

Appropriate timeframes, based on the first round of charter reviews, would seem to be ~1 month for (2) and maybe 3 days for (3).

### LINKED ISSUE #36: Charter Template: Document reuse potential in Description section
**Notes from PM F2F (9/25/18):**

Add a note to the *Description* to document whether a project is HCA specific or has reuse in other communities - *Data Store FISMA scenario* for example.

### LINKED ISSUE #41: Rename charter-proposed and charter-final-review
- [x] Rename *charter-proposed* to *charter-community-review*
 - [x] Rename *charter-final-review* to *charter-oversight-review*
 - [x] Clarify that *charter-oversight-review* is limited to oversight reviewers and not the general community

### PR DESCRIPTION
**Pull request by brianraymor**
Closes #29 
Closes #36 
Closes #41 

If a charter is approved, then ensure that the charter owners are included in the PM/Architecture mailing lists and slack channels.

Editorial updates to match _RFC style_, including
* Outline matches process flow
* Included examples

### CODE REVIEW

File: charters/README.md [hunk 1]
```diff
@@ -1,26 +1,140 @@
...
-  **For charters with software deliverables:**
+- Submit a pull request. Add the "charter-community-review" label
+
+- Add a minimum two week _last call_ for the completion of the Community review to the top-level summary comment in the charter pull request.
+
+    **EXAMPLE** 
+
+    **April 1**: Last call for community review
+
+- Request a Community review of the charter on the HumanCellAtlas **#dcp** slack channel. Include a link to the charter pull request and the last call deadline:
```
Comment (briandoconnor) [inline charters/README.md:34]: You might also want them to always include a link to this process document so people always have this process handy when they see the review request :-)

File: charters/README.md [hunk 2]
```diff
@@ -1,26 +1,140 @@
...
+
+- At the *DCP PM* or *DCP Architecture* meeting, review the summary comment. If there is rough consensus to reject the charter, replace "charter-oversight-review" with "charter-rejected", and close the pull request with the rationale in the comment.
+
+- Announce the rejection of the charter on the HumanCellAtlas #dcp slack channel. Include a link to the charter pull request:
+
+  **EXAMPLE**
+
+  ***@channel**: DCP Architecture declined the Data Store charter - https://github.com/HumanCellAtlas/dcp-community/pull/8*
+
+## Appealing Charter Decisions
```
Comment (briandoconnor) [inline charters/README.md:134]: Great, thanks for including

File: charters/charter-template.md [hunk 3]
```diff
@@ -4,7 +4,8 @@
 
 ## Description
 
-_Describe the vision of the project in **2-3** sentences. Consider this section as your "elevator pitch" to new community members._
+_Describe the vision of the project in **2-3** sentences. Consider this section as your "elevator pitch" to new community members. For software charters, indicate whether the project is specific to HCA DCP or has the potential for reuse by other communities: "The Data Store is designed to be reused in multiple projects, with HCA being the first user."_
```
Comment (briandoconnor) [inline charters/charter-template.md:7]: 👍

**Review by briandoconnor — APPROVED** [review]

**Review by lauraclarke — APPROVED** [review]
Looks great to me, I have a couple of documentation notes about DCP team organisation which I will create issues for as they don't require any changes to the text here

**Pull request merged**